wiki/Multi/en.md

```diff
@@ -0,0 +1,487 @@
+
+#### Interface
+
+_Only available in osu! mode. **UNRANKED**_
+
+![Tag Coop Interface][Tag Coop Interface image]
+
+_Tag Coop interface_
+
+![Tag Team Vs Interface][Tag Team Vs Interface image]
+
+_Tag Team Vs interface_
+
+Tag Coop makes all players in the Match Setup work together to pass the beatmap, one combo at a time.
+Tag Team Vs is just Tag Coop with red/blue team competition.
+
+Do note that this team mode is only available in **osu! only** and **UNRANKED** (meaning, it will not be counted to ranked score).
+
+In this mode, player placement is important as it will determine when it is the player's turn.
+All players shares a health bar / Each team has its own respective health bar that are visible to teammates only.
+Unlike other team modes, failing the beatmap means returning to the Match Setup immediately without results / giving victory to the other team.
+
+An arrow will show the player who is currently playing the beatmap's combo.
+The name will be shown at the lower right of the screen.
+There will be warning arrows when it is the player's turn and the combo will be in colour until the combo ends.
+The player can adjust for its own personal combo colour at Match Setup under Tag Colour.
+
+**All players must spin the spinner**.
+Refusal or unable to complete the spinner will break the current combo count.
+
+If a player quits during the match, [Auto][Auto wikilink] will take over for the player that left.
+
+Players using [Sudden Death (SD)][Sudden Death wikilink]/[Perfect (PF)][Perfect wikilink] will have a different interface than other players, especially when it comes to the effects.
+If the teammate does not use that mod and made a small mistake enough to trigger the mod effect, the beatmap will continue as normal, and players using the mod will have their health bar interface drained and stuck to empty with no ill effect (meaning, visual effect only).
+If the user _using_ the mod made a mistake and triggered it, the actual effect will occur, and failed state will be triggered immediately.
+
+#### Result
+
+![Tag Coop Grade screen][Tag Coop Grade image]
+
+_Tag Coop grade screen_
+
+![Tag Team Vs Grade screen][Tag Team Vs Grade image]
+
+_Tag Team Vs grade screen_
+
+As stated before, score will not be counted for ranked score and performance points calculation because it is an **UNRANKED** play.
+
+For Tag Coop, the result shown is the total score, hit score and accuracy of the team.
+Individual results will not be shown, and can only be seen in Match History.
+
+For Tag Team Vs result screen, refer to Team Vs's result screen explanation.
+
+## Trivia
+
+### History
+
+- Match Setup used to only allows 2 up to a total of 8 players only.
+- osu!mania game mode was noted as `3` in Lobby for players that have not yet updated their _osu!_ to the latest release patch.
+- The only available game modifier allowed for players in a Match Setup used to be the No Video mod before the mod was deprecated in favour of Visual Settings.
```

Comment:
Could rephrase to: `No Video was the only game modifier that was allowed for players to use; however, it was removed in favour of its placement in the Visual Settings.`